src/components/layouts/PublicPageLayout/PublicPageLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Grid, WithStyles, withStyles } from '@material-ui/core';
+
+import styles from './styles';
+
+export interface PublicPageLayoutProps extends WithStyles<typeof styles> {
+  children: JSX.Element;
+}
+
+function PublicPageLayout(props: PublicPageLayoutProps): JSX.Element {
+  const { children, classes } = props;
+  return (
+    <Grid
+      className={classes.root}
+      container
+      direction='row'
```

Comment:
Just curious as to why you picked direction row here. Is it so that the whole screen is utilized, at least horizontally, if there are multiple children? Or does it just make sense from a design standpoint?

Reply:
great question
idk
it looks fine and will be encapsulated anyways :)